src/components/CanvasAnnotations.js

```diff
@@ -75,6 +75,7 @@ export class CanvasAnnotations extends Component {
                 component={listContainerComponent}
                 className={classes.annotationListItem}
                 key={annotation.id}
+                annotation={annotation}
```

Comment:
Could you provide an example of how a plugin will pick up this data?

Reply:
Sure. For a component that wraps `CanvasAnnotations` they can enhance those props by specifying a `listContainerComponent`. https://github.com/ProjectMirador/mirador-annotations/blob/master/src/plugins/canvasAnnotationsPlugin.js#L18

That `listContainerComponent` would then be able to take advantage of this. Maybe this is all just too much indirection?

Reply:
Got it; i think it's fine, just some magic in `ListItem` that's new to me.